File: src/Server/Modules/Catalog/Modules.Catalog.Core/Entites/Product.cs
using FluentPOS.Shared.Abstractions.Domain;

namespace FluentPOS.Modules.Catalog.Core.Entites
{
    public class Product : BaseEntity //IAuditableEntity
    {
        public string Name { get; set; }
        public string LocaleName { get; set; }
        public int BrandId { get; set; }
        public virtual Brand Brand { get; set; }
        public decimal Price { get; set; }
        public decimal Cost { get; set; }
        public string ImageUrl { get; set; }
        public string Tax { get; set; }
        public string TaxMethod { get; set; }
        public string BarcodeSymbology { get; set; }
        public decimal AlertQuantity { get; set; }
        public string Detail { get; set; }
    }
}

File: src/client/src/app/modules/pos/models/product.ts
export interface Product {
  id: string;
  name: string;
  localeName: string;
  brandName: string;
  categoryName: string;
  price: number;
  cost: number;
  imageUrl: string;
  tax: number;
  taxMethod: string;
  barcodeSymbology: string;
  isAlert?: boolean;
  alertQuantity: number;
  detail: string;
}
